frontend/components/editions/EditionDeletionPopup.tsx

```diff
@@ -0,0 +1,101 @@
+import { Dispatch, FC, SetStateAction, useState } from 'react';
+import Popup from 'reactjs-popup';
+import { SpinnerCircular } from 'spinners-react';
+
+type EDPProps = {
+  /**
+   * function that deletes the edition in the backend service
+   *
+   * @see {@link EDITIONS_PAGE}
+   */
+  deleteEdition: () => Promise<void>;
+
+  /**
+   * state that holds wether the deletion confirm should be shown or not
+   */
+  openDeleteForm: boolean;
+
+  /**
+   * state update function that sets wether the deletion confirm should be shown or not
+   */
+  setOpenDeleteForm: Dispatch<SetStateAction<boolean>>;
+};
+
+/**
+ *
+ * @param EDPProps - Edition Deletion Popup Props
+ * @returns Edition Deletion Popup Component
+ */
+const EditionDeletionPopup: FC<EDPProps> = ({
+  deleteEdition,
+  openDeleteForm,
+  setOpenDeleteForm,
+}: EDPProps) => {
+  const [loading, setLoading] = useState(false);
+
+  const closePopup = () => {
+    setOpenDeleteForm(false);
+  };
+
+  const doDelete = async () => {
+    setLoading(true);
+    await deleteEdition();
+    setLoading(false);
+    closePopup();
+  };
+
+  return (
+    <Popup
+      modal
+      open={openDeleteForm}
+      data-backdrop="static"
+      data-keyboard="false"
+      closeOnDocumentClick={false}
+      lockScroll={true}
+      position="center center"
+    >
+      <div className="modal max-w-screen flex max-h-[85vh] min-w-[600px] flex-col bg-osoc-neutral-bg py-5">
+        <a
+          className="close"
+          onClick={(e) => {
+            e.stopPropagation();
+            closePopup();
+          }}
+        >
+          &times;
```

Comment:
What does this `&times` trick do?

Reply:
This is the HTML code for a `multiply` icon, which is the cross on the close button in the top right.